spec/beaker/dsl/install_utils/foss_utils_spec.rb

```diff
@@ -414,7 +414,7 @@ def logger
     context 'on el-5' do
       let(:platform) { Beaker::Platform.new('el-5-i386') }
       it 'installs' do
-        expect(subject).to receive(:on).with(hosts[0], /puppetlabs-release-el-5\.noarch\.rpm/)
+        expect(hosts[0]).to receive(:install_package_with_rpm).with('http://yum.puppetlabs.com/puppetlabs-release-el-5.noarch.rpm', '--replacepkgs', {:package_proxy=>false})
```

Comment:
Sorry, one of us is misunderstanding something; it could be me!

My spec test is saying: *I expect that when the `install_puppet` method is called, an options hash `{:package_proxy=>false}` will be passed to the `install_package_with_rpm` method*.  I am not saying that the `install_package_with_rpm` requires an options hash to be passed (it doesn't).  I am testing the behaviour of `install_puppetlabs_release_repo_on` here; not the `install_package_with_rpm` method.

Now you could question the value of this test; I also question the value of this test and some of the other tests in this class; but I have only updated a test that I inherited after I refactored the code.

Note that I have not provided a test of the `install_package_with_rpm` method at all.  The reason is there were no tests for the [install_package](https://github.com/puppetlabs/beaker/blob/master/spec/beaker/host/unix/pkg_spec.rb#L128) method for me to copy. :)

If you feel I really must add tests for my method (it doesn't do much other than call `execute` which is tested elsewhere) let me know.